sway/sway-ipc.7.scd

```diff
@@ -0,0 +1,1579 @@
+sway-ipc(7)
+
+# NAME
+
+sway-ipc - IPC protocol for sway
+
+# DESCRIPTION
+
+This details the interprocess communication (IPC) protocol for *sway*(1). This
+IPC protocol can be used to control or obtain information from a sway process.
+
+The IPC protocol uses a unix socket as the method of communication. The socket
+path is _$XDG\_DESKTOP\_DIR/sway-ipc.$UID.<PID>.sock_, where _<PID>_ is the PID
+of the sway process. The path is stored in the environment variable _SWAYSOCK_
+and, for backwards compatibility with i3, _I3SOCK_. You can also retrieve the
+socket path by calling _sway --get-socketpath_.
```

Comment:
I would rather not document the name of the socket in case we want to change it later (and I might want to). Let's just say it'll be in the `SWAYSOCK` variable (and `I3SOCK`).

Reply:
Changed